src/hmem.c

```diff
@@ -44,6 +44,7 @@ struct ofi_hmem_ops {
 	int (*cleanup)(void);
 	int (*copy_to_hmem)(void *dest, const void *src, size_t size);
 	int (*copy_from_hmem)(void *dest, const void *src, size_t size);
+	bool (*is_hmem_iface)(const void *addr);
```

Comment:
The commit message could really be expanded to describe what is going on.  It took me several minutes to figure out what this call is actually supposed to do, as the name is misleading.  The call is checking if the provided address belongs to the specific hmem type, not if it is an hmem_iface.

Maybe rename to is_addr_valid()?

Reply:
I'll update the commit message and this operation.